### DISCUSSION TITLE
[@types/node-telegram-bot-api] Missing `sendPhoto` formal parameter.

### DISCUSSION THREAD

**Discussion by CoeJoder** [DefinitelyTyped/DefinitelyTyped#58524] [Issues with a @types package]
The `fileOptions` parameter of [sendPhoto()](https://github.com/yagop/node-telegram-bot-api/blob/release/doc/api.md#TelegramBot+sendPhoto) is missing from the declared types.

[a comment by an automated account is not shown]

**Comment by martin-badin** [comment]
Hi @CoeJoder. PR is ready. Please could you review it? #58625
